### PR TITLE
quick fix for service.name scripted field

### PR DESCRIPTION
**Pull request by graphaelli**
for elastic/kibana#27333

### CODE REVIEW

File: model/error/_meta/fields.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   fields:
     - name: view errors
       type: keyword
-      script: "doc['context.service.name'].value"
+      script: "doc['service.name'].size() > 0 ? doc['service.name'].value : doc['context.service.name'].size() > 0 doc['context.service.name'].value : null"
```
Comment (sqren) [inline model/error/_meta/fields.yml]: Doesn't this have a typo in the second ternary?

Shouldn't it be (notice the added `?`):
```suggestion
      script: "doc['service.name'].size() > 0 ? doc['service.name'].value : doc['context.service.name'].size() > 0 ? doc['context.service.name'].value : null"
```

**Comment by graphaelli** [comment]
jenkins, please test this

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
gh webhook seems to have failed, this passed [ci](https://apm-ci.elastic.co/job/elastic+apm-server+pull-request/444/) except for the expected sync job failure

**Pull request merged**